code/game/objects/items/AI_modules.dm

```diff
@@ -23,11 +23,15 @@ AI MODULES
 	var/list/laws = list()
 	var/bypass_law_amt_check = 0
 	custom_materials = list(/datum/material/gold = 50)
+	var/key_override = FALSE
+	var/law_override = FALSE
 
 /obj/item/aiModule/examine(var/mob/user as mob)
 	. = ..()
 	if(Adjacent(user))
 		show_laws(user)
+	if(key_override)
+		desc += "\nInput upload code 666 to override upload restriction."
```

Comment:
1. examin() is not proper proc for this. there is explanation text proc.
2. `desc += "text"` is not correct. This will add explanation to `desc` everytime you examine this.
3. It should warn people like `Once uploaded, the bypass code 666 will be disabled from this module.`